requirements.txt

```diff
@@ -5,4 +5,4 @@ psutil>=5
 scikit-learn>=0.20.0
 fastdtw
 setuptools>=40.1.0
-
+qiskit~=0.26.0
```

Comment:
This is not required. There's `terra` dependency on the first line.

Reply:
Thank you! I'll remove that one.